compiler-rt/cmake/config-ix.cmake

```diff
@@ -751,7 +751,8 @@ else()
   set(COMPILER_RT_HAS_ASAN FALSE)
 endif()
 
-if (COMPILER_RT_HAS_SANITIZER_COMMON AND RTSAN_SUPPORTED_ARCH)
+if (COMPILER_RT_HAS_SANITIZER_COMMON AND RTSAN_SUPPORTED_ARCH AND
+    OS_NAME MATCHES "Android|Darwin|Linux")
```

Comment:
@cjappl how about this and let motivated folks with access to those OSes to test and enable when needed.

Reply:
Sounds good to me.

I believe we need to add the conditional check for COMPILER_RT_HAS_RTSAN in compiler-rt/lib/rtsan/CMakeLists.txt. We missed that in the first PR.

With that change, this should be good! I'm going to approve, assuming that you add that conditional in there.

Reply:
I actually rather limit to whatever you are willing to work in near future.
What is your platforms of interest?

Reply:
Linux and Darwin are the platforms that we have easiest development access to in ubuntu and darwin. I think starting there and building up would be just fine.

Reply:
> I believe we need to add the conditional check for COMPILER_RT_HAS_RTSAN in compiler-rt/lib/rtsan/CMakeLists.txt. We missed that in the first PR.

COMPILER_RT_HAS_${} checked by root lib/ and test/ CMakeLists